gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -33,6 +26,43 @@
         "minimum": 0,
         "maximum": 10000
       }
+    },
+    "required": [
+      "auth_type"
+    ],
+    "dependencies": {
+      "auth_type": {
+        "oneOf": [{
+          "properties": {
+            "auth_type": {
+              "describe": "Get the client and the introspection end point from the oidc issuer endpoint",
+              "enum": ["oidc_issuer_endpoint"]
+            }
+          }
+        }, {
+          "properties": {
+            "auth_type": {
+              "describe": "Specify the client ID and the client secret",
```

Comment:
This probably should say something like:

> Specify the Token Introspection Endpoint, Client ID, and Client Secret.

Reply:
Changed.